index.bs

```diff
@@ -2293,35 +2293,72 @@ The W3C Council</h4>
 	the [=Chair of the Council=] <em class="rfc2119">should</em> dissolve the [=Council=]
 	and call for a new one to be convened.
 
-Issue: The following text about recusals is does not relfect the latest thinking.
-However, the latest thinking isn't fully stabilized yet,
-and thus not ready to replace this.
-See <a href="https://github.com/w3c/w3process/issues/278">https://github.com/w3c/w3process/issues/278</a>
-and in particular <a href="https://github.com/w3c/w3process/issues/278#issuecomment-995868961">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>,
-as well as <a href="https://docs.google.com/document/d/1lB8Q_pvqxfN204FQHZ8D2HObxfo36qnguif9mGpEJTU/edit#">https://github.com/w3c/w3process/issues/278#issuecomment-995868961</a>.
-
-	No one shall be both a party and an adjudicator.
-	The person making either the decision being objected to or appealed,
-	and the person making the objection or appeal,
-	<em class=rfc2119>must</em> [=recuse=] themselves.
-	Participants <em rfc=2119>should</em> [=recuse=]
-	if they believe their participation would compromise the integrity of the Council decision,
-	and <em class=rfc2119>may</em> do so at any time.
-	However, recusal is disqualification from participation,
+	Individual recusal is not used in the Council. 
+	The potential council decide themselves, uniformly, which reasons for each potential member to serve 
+	rise to a sufficient level for that potential member to be [=dismissed=].
+	No-one is automatically dismissed.
+				
+	The team assignee <em class=rfc2119>must</em> ensure a list is developed of all potential council members, 
+	and mark against each one possible reasons that they might need to be dismissed. 
+	The W3C community,
+	including members and team, and potential council members, 
+	<em class=rfc2119>must</em> be given an opportunity to contribute possible reasons to this list.
+
+	Example reasons (none of which are diagnostic) could include:
+	<ol>
+		<li>
+			They filed the formal objection, helped author it, 
+			or work for the same organization as the author/filer
+		<li>
+			They made the decision being objected to
+		<li>
+			They were an editor of, or a major contributor to, the document in question
+		<li>
+			Their business depends on the question (in either direction)
+		<li>
+			They hold sufficiently strong pre-existing opinions in either direction 
+			(e.g. a well-known proponent or objector) that their ability 
+			to be open-minded may reasonably be in doubt
```

Comment:
```suggestion
```

People who are elected to the TAG or AB are either considered by those who voted for them sufficiently able to be open-minded, or (less likely, given they get there through STV so typically have to convince a segment of those who preferred someone else) their super strongly-held beliefs are considered OK.

In any event having the council vote on who holds a belief so strongly that the rest of the council doesn't think they should express it formally where it matters strikes me as a particularly dangerous precedent.

In particular, a number of recent objections have turned on _inter alia_ whether something is in the scope of W3C. This issue seems to have strongly held beliefs on both sides from many key stakeholders. Dismissing them  from the council seems a pathway to losing important voices, not dismissing them begs the question of how to justify dismissing people for other strongly-held beliefs.

Reply:
This was long-debated, and we mostly agree I think. It's not that they have opinions (who doesn't?) it's that for them on this subject the opinions are more on the level of a belief that blinkers their ability to to be open-minded. This rarely happens, IMHO. But people will want to say "so and so has always maintained that X is a great idea, they should be dismissed" and the potential Council should consider whether they are open-minded or not

Reply:
We're setting people an impossible problem if we are asking them to understand within someone else's mind how open they are to change. They might persuasively appear to be open-minded whilst privately having a strong intention not to change their minds, or appear to have quite definite views but still in fact be open to change if the evidence is compelling.

Requiring as part of the Process that people correctly solve this impossible problem for each potential Council member seems unwise.

Reply:
It's not that they have opinions; it's that the rest of the Council feels/perceives that the result will be tainted in the eyes of the community if that person serves for this question. We give the entire potential Council the opportunity to dismiss potential members if they feel that the result would be tainted by that person's presence. So they are judging a community perception, not the mind of the person.

Reply:
It's not obvious that extending the requirement to mind-read to a larger group of subjects will lead to greater accuracy. If the requirement is to allow the community to express objections then we should be adding an explicit opportunity for that to occur, possibly gated by strict requirements on the permitted reasons for such an objection.

Reply:
I see what you're saying @dwsinger but it still feels very odd: the potential council is being asked to take a moderator role in reviewing the list of objections, received from the community, to each member of that same potential council. Even though no individual can assess the objections made to their own participation, my flawed mind-reading and future-telling skills suggest that there will nevertheless be a perception that the potential council is a club of people who know each other well and work with each other regularly and are therefore generally well-disposed to each other, biasing against the objectors.

The very process of assessing the objections is likely not to seem independent and fair. 

I can't see a solution to that except to have some separate group of people who are already unable to participate in the FO Council: it would be that group's job to process the objections. Possibly there are other better models?